digitalocean-1-click/image.pkr.hcl

```diff
@@ -0,0 +1,40 @@
+variable "package_version" {
+  type    = string
+  default = "1-rc3"
+}
+
+variable "token" {
+  type      = string
+  sensitive = true
+  default   = env("DIGITALOCEAN_ACCESS_TOKEN")
+}
+
+source "digitalocean" "x" {
+  api_token     = "${var.token}"
+  image         = "ubuntu-18-04-x64"
+  region        = "sfo3"
+  size          = "s-1vcpu-2gb"
```

Comment:
Did you test on "s-1vcpu-1gb"?  Ideally we'd like to be able to fit into the smallest droplet.